dashboard/app/controllers/student_work_evaluations_controller.rb

```diff
@@ -15,6 +15,29 @@ def create
     end
   end
 
+  # GET /student_work_evaluations/:userId/:levelId/:unitId
+  def get_student_work_evaluations
+    user_id = params[:user_id]
+    level_id = params[:level_id]
+    unit_id = params[:unit_id]
+
+    evaluations = StudentWorkEvaluation.where(
+      student_id: user_id,
+      level_id: level_id,
+      unit_id: unit_id
+    )
```

Comment:
I think we want to be more specific about the type of evaluation we want. This, as is, could return a mix of UserLevelEvaluations and UserLevelSkillEvaluations. In this case, we want `get_most_recent_user_level_evaluation` ?

Reply:
Good catch!  Fixed!